### DISCUSSION TITLE
Better anatomy

### DISCUSSION THREAD

**Discussion by janstehno** [janstehno/cotw-companion#38] [Ideas]
You would have an option to edit opacity of organs or the silhouette. I could also add something new (bones for example) to the anatomy image as well. There is also suggestion to add trophy organ indicator.

[original discussion](https://github.com/janstehno/cotwcompanion/discussions/34)

**Comment by JeanRoxus** [comment]
Perhaps combine with this the possibility of visualizing what an animal would look like with the different types of coats available and their percentage of appearance.

**Reply by janstehno** [reply]
I would rather do another section with horizontal slider to show the fur types.
